monkey/agent_plugins/credentials_collectors/chrome/src/chrome_credentials_collector.py

```diff
@@ -0,0 +1,63 @@
+import time
+from typing import Sequence
+
+from common.agent_events import CredentialsStolenEvent
+from common.credentials import Credentials
+from common.event_queue import IAgentEventPublisher
+from common.tags import (
+    CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+    UNSECURED_CREDENTIALS_T1552_TAG,
+)
+from common.types import AgentID, Event
+
+from .typedef import CredentialsDatabaseProcessorCallable, CredentialsDatabaseSelectorCallable
+
+CHROME_CREDETIALS_COLLECTOR_TAG = "chrome-credentials-collector"
+CHROME_COLLECTOR_EVENT_TAGS = frozenset(
+    (
+        CHROME_CREDETIALS_COLLECTOR_TAG,
+        DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+        UNSECURED_CREDENTIALS_T1552_TAG,
+        CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    )
+)
+
+
+class ChromeCredentialsCollector:
+    def __init__(
+        self,
+        agent_id: AgentID,
+        agent_event_publisher: IAgentEventPublisher,
+        select_credentials_database: CredentialsDatabaseSelectorCallable,
+        process_credentials_database: CredentialsDatabaseProcessorCallable,
+    ):
+        self._agent_id = agent_id
+        self._agent_event_publisher = agent_event_publisher
+        self._select_credentials_database = select_credentials_database
+        self._process_credentials_database = process_credentials_database
+
+    def run(self, interrupt: Event) -> Sequence[Credentials]:
+        database_paths = self._select_credentials_database()
+
+        timestamp = time.time()
+        credentials = self._process_credentials_database(database_paths)
+
+        if len(credentials) > 0:
```

Comment:
If we attempted to access any databases, we should report an event, since this is potentially observable. It maybe the case that the event has no credentials in it (i.e. we weren't successful), but we did try.
```suggestion
        if len(databse_paths) > 0:
```